### PR TITLE
ADD description on payment mode and show it on invoice report

### PR DESCRIPTION
**Pull request by angelmoya**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You have to check my comments on https://github.com/OCA/bank-payment/pull/185

**Comment by angelmoya** [comment]
Ok, I checked your comments second time, and had tabs in xml file, so I did a commit and squash it. I think now it's ok.

File: account_banking_payment_export/i18n/es.po [hunk 1]
```diff
@@ -92,9 +77,20 @@ msgid "ID"
...
 
 #. module: account_banking_payment_export
+#: model:ir.model,name:account_banking_payment_export.model_account_invoice
+msgid "Invoice"
+msgstr "Factura"
+
+#. module: account_banking_payment_export
+#: view:account.move.line:account_banking_payment_export.payment_order_populate_view_move_line_tree
+msgid "Journal Entry"
+msgstr "Journal Entry"
```
Comment (JordiBForgeFlow) [inline account_banking_payment_export/i18n/es.po]: Falta traducir este término por "Asiento contable"

File: account_banking_payment_export/i18n/es.po [hunk 2]
```diff
@@ -214,6 +226,12 @@ msgid "Specify the Code for Payment Type"
 msgstr "Especifica el código para el tipo de pago"
 
 #. module: account_banking_payment_export
+#: code:addons/account_banking_payment_export/models/account_invoice.py:34
+#, python-format
+msgid "Structured Reference"
+msgstr "Structured Reference"
```
Comment (JordiBForgeFlow) [inline account_banking_payment_export/i18n/es.po]: Traducir "Structured Reference" a "Referencia Estructurada"

**Comment by JordiBForgeFlow** [comment]
Reviewed. Other than the comments on translations, looks :+1: to me.

**Comment by angelmoya** [comment]
Ok, I just fixed this commit.

**Pull request merged**